client/views/flashcards/Flashcard.js

```diff
@@ -0,0 +1,18 @@
+Template.Flashcard.events({
+  "click .flip": function(event, template){
+    template.$('.front').toggleClass('hidden');
+    template.$('.back').toggleClass('hidden');
+  }
+});
+
+Template.Flashcard.helpers({
+  indexIsZero: function() {
+    return this.index == 0
+  }
+});
+
+UI.registerHelper('addIndex', function (all) {
```

Comment:
What is this for? I can't find anything in the Meteor docs for this. When I removed it everything is still working...